src/java/org/apache/cassandra/tools/NodeTool.java

```diff
@@ -105,7 +105,6 @@ public int execute(String... args)
                 Compact.class,
                 CompactionHistory.class,
                 CompactionStats.class,
-                CreateEpochUnsafe.class,
```

Comment:
may you rest in pieces... you got us to where we are today...  a moment of silent please....